src/Crownpeak/Yves/FirstSpiritContent/Plugin/Twig/AttributesTwigFunction.php

```diff
@@ -24,7 +24,7 @@ class AttributesTwigFunction extends AbstractPlugin implements TwigPluginInterfa
 
     /**
      * This is the name of the global function that will be available in the twig templates.
-     * usage: {{ firstSpiritAttributes(id, type, template, title, locale) }}
+     * usage: {{ firstSpiritAttributes(id, type, template, title, locale [,isFsDriven]) }}

```

Comment:
```suggestion
     * usage: {{ firstSpiritAttributes(id, type, template, title, locale, [isFsDriven]) }}
```

Reply:
I assume it should also be `isFsDriven` and not `[isFsDriven]` or is this intentional?